pages/common/lambo.md

```diff
@@ -0,0 +1,36 @@
+# lambo
+
+> A super-powered `laravel new` for Laravel and Valet.
+> More information: <https://github.com/tighten/lambo>.
+
+- Create a new Laravel application:
+
+`lambo new {{name}}`
```

Comment:
Maybe this would aid clarity? Same goes for the rest of the examples.
```suggestion
`lambo new {{app_name}}`
```

Reply:
Yeah, I did think about that. I was just being consistent with other pages.

Reply:
Ah, ok. Still, I think this adds enough clarity to consider breaking consistency.

I guess it doesn't really matter then.

Reply:
I'll update this shortly. Cheers.